encrypted-media-respec.html

```diff
@@ -609,19 +609,24 @@
               <p>
               Supported configurations, including supported audio and video codecs, may depend on availability of optional capabilities such as
               <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> and persistent state. The following algorithm iteratively tries to find a configuration
-              that is both supported and has user consent, each iteration considering a smaller set of possible configurations.
+              that is both supported and has user consent (or does not need consent).
               </p>
               <p>
               User Agents should reuse earlier consent responses, when appropriate, at least for the duration of the <a def-id="requestMediaKeySystemAccess"></a>
               algorithm in order to avoid repeated requests to the user for similar configurations.
               </p>
+              <p>
+                The variable <var>restrictions</var> in the steps below represents the configurations for which consent has been denied. It is used to determine
+                whether user consent for a candidate configuration or partial configuration has been denied. Consent is denied for a partial configuration if every derived
```

Comment:
I'm not sure about this. "... would be denied ..." implies a prediction about what the user _would_ say if we asked them, and we cannot know that (they might change their mind). We've explained above that a single consent prompt can be interpreted as denying a whole slew of configurations based on exactly what the user was asked and how they responded so I think it's reasonable to say _... consent has been denied ..._.

Would it help to say _... consent has been denied during the execution of this algorithm or based on persisted consent information for the origin._ ?